### ISSUE TITLE
Enhancement: Scroll Bars

### ISSUE THREAD

**Issue by KelsiElshaw** [ApexaCorp/feedback-portal#69]
It would be helpful if the scroll bars in APEXA were more visible (especially the scroll bar on the navigation panel).

**Comment by Bobbisue-HUB** [comment]
I completely agree - the 'white' bar is not visible until selected so difficult when navigating initally

**Comment by Angela-MLI** [comment]
Definitely!  I could see this being an issue very quickly with users.

I also find the scroll function does not work using my mouse scroll wheel.  Is this included in this fix or do I need to log this as a separate issue?  Thanks!

**Comment by jromero-PPI** [comment]
I have the same issue with Angela - mouse not scrolling. I wasn't sure if this is a system issue since others seem to be able to do this. Maybe compatibility issue?
